Test/Integration/Indexing/Config/MultiStoreConfigTest.php

```diff
@@ -88,15 +88,17 @@ public function testMultiStoreIndicesCreation()
 
         $this->indicesConfigurator->saveConfigurationToAlgolia($fixtureSecondStore->getId());
 
-        $defaultCategoryIndexSettings = $this->algoliaHelper->getSettings(
+        $defaultCategoryIndexOptions = $this->indexOptionsBuilder->buildWithEnforcedIndex(
```

Comment:
Can we compute here as well? Or would that not work with these fixtures?

Reply:
I think the rework of index names in the integration tests should be done on a separate ticket since it needs some tidying, I've created a ticket for it (MAGE-1298)